plugin/evm/orderbook/config_service.go

```diff
@@ -18,7 +18,9 @@ type IConfigService interface {
 	GetActiveMarketsCount() int64
 	GetUnderlyingPrices() []*big.Int
 	GetLastPremiumFraction(market Market, trader *common.Address) *big.Int
+	GetLastPremiumFractionAtBlock(market Market, trader *common.Address, blockNumber uint64) *big.Int
```

Comment:
should we remove these *AtBlock functions since we can't get the state at all the past blocks?

Reply:
yes done.